apps/nextjs/src/components/AppComponents/Chat/chat-start-form.tsx

```diff
@@ -59,19 +59,16 @@ export function ChatStartForm({
         />
         <div className="absolute bottom-10 right-10 top-10 flex items-center justify-center ">
           <Tooltip>
+            <LoadingWheel visible={isSubmitting} />
```

Comment:
Fixes a blank state where the loading wheel icon has not lazy loaded

Reply:
Ah, that's a nice approach